tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -123,6 +123,24 @@ public interface ICollectionItem
                 .x(() => fakes.Should().OnlyContain(item => Fake.GetFakeManager(item) != null));
         }
 
+        [Scenario]
+        public void InterfaceWithAlikeGenericMethod()
+        {
+            "Given an interface with methods having multiple generic arguments that can be inheirited"

```

Comment:
"inheirited" is misspelled, but I think it doesn't add much anyhow. Perhaps something like